code/game/machinery/doors/firedoor.dm

```diff
@@ -142,6 +142,14 @@
 	if(density)
 		open()
 	else
+		if(iscarbon(user))
+			var/mob/living/carbon/C = user
+			if(C.has_trait(TRAIT_PACIFISM))
+				var/T = get_turf(src)
+				for(var/mob/living/L in T)
+					if((L.stat != DEAD) && !L.has_trait(TRAIT_FAKEDEATH))

```

Comment:
i was a touch drunker when i typed this up, you can remove the fakedeath check. wouldn't really be a concern.